spec/index.bs

```diff
@@ -87,26 +83,26 @@ figure.table .overlarge {
+This specification provides a mechanism for the use of verifiable credentials in a highly scalable, secure, and verifiable way. A large part of the integrity of a verifiable credential is how to structure the credential so that all three parties — issuer, holder, verifier — may have a singular mechanism of trust into what they are using. We call that document a [=Credential Schema=].
 
-This specification provides a standardized way of creating [=Credential Schema=]s to be used in credentialing platforms, how to version them, and how to read them.
+This specification provides a standardized way of creating [=Credential Schemas=] to be used in credentialing platforms, how to version them, and how to read them. Credential Schemas may apply to any portion of a Verifiable Credential. Multiple JSON Schemas may back a single Verifiable Credential, e.g. a schema for the `credentialSubject` and another for other credential properties.
 
 # Terminology # {#terminology}
 
 : <dfn>id</dfn>
-:: A globally unique identifier to locate the schema on a distributed ledger. Each credential schema has its own unique identifier, and each version of a credential schema is required to have its own unique identifier. <br/> <br/> This identifier is a [Generic DID Parameter Name](https://w3c.github.io/did-core/#did-syntax) based upon the author of the schema. For example, if the author had a did like <pre>did:work:abcdefghi</pre> a possible schema ID the author created would have an identifier such as: <pre>did:work:abcdefghi;id=17de181feb67447da4e78259d92d0240;version=1.0</pre>
+:: A locally unique identifier to locate the schema on a given data storage medium (e.g. a database, ledger, distributed file store). Each credential schema has its own unique identifier, and each version of a schema is required to have its own unique identifier. <br/> <br/> This identifier is a [Uniform Resource Identifier](https://www.rfc-editor.org/rfc/rfc3986) that <i>may</i> contain informatoin pertaining to the author and version of the schema. For example, if the author had a DID like <pre>did:key:abcdefghi</pre> a possible schema ID the author created would have an identifier such as: <pre>did:key:abcdefghi;id=17de181feb67447da4e78259d92d0240;version=1.0</pre>.
 
 : <dfn>Credential Schema</dfn>
 :: The data template for a credential. Refers to the entirety of a [=Credential Schema=], including both the [=Metadata=] and [=JSON Schema=]. The term may refer to a document either with, or without a [=proof=].
 
 <pre class="example" highlight="json">
 {
-  "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/1.0/schema.json",
-  "modelVersion": "1.0",
-  "id": "did:work:MDP8AsFhHzhwUvGNuYkX7T;id=06e126d1-fa44-4882-a243-1e326fbe21db;version=1.0",
+  "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/2.0/schema.json",
+  "version": "2.0",
+  "id": "did:key:MDP8AsFhHzhwUvGNuYkX7T;id=06e126d1-fa44-4882-a243-1e326fbe21db;version=1.0",
```

Comment:
these don't look like valid did keys to me... https://did.key.transmute.industries/

Reply:
will change to example